src/ParseQuery.js

```diff
@@ -1313,6 +1315,10 @@ class ParseQuery {
   /**
    * Includes nested Parse.Objects for the provided key.  You can use dot
    * notation to specify which fields in the included object are also fetched.
+   * 
+   * If you want to include all nested Parse.Objects pass in '*'
+   * <pre>query.include('*');</pre>
```

Comment:
perhaps we need to put a version number here?

Reply:
Which version? 2.1.0?

Reply:
I mean server version?

Reply:
I have no idea mate, I can try to backtrack

Reply:
Ok ok, so basically it already works if you pass include All in the rest SDK.

Reply:
Yes that is correct. Basically this won't work until the next release unless I re-add includeAll to this SDK

Reply:
Yeah, let’s just use the * notation then we’ll perhaps be able to expand on with reg ex etc...

Reply:
❤️

Reply:
How best can we update the guides simultaneously with the API docs